### DISCUSSION TITLE
Occasionally get served plain text version of website on DigitalOcean

### DISCUSSION THREAD

**Discussion by tollanes** [vercel/next.js#54596] [App Router] [answered]
Hi, after upgrading to Next.js 13 we have been seeing some weird behavior every now and then. We are using the localization system defined in the documentation on the app directory section, the [lang] sub-route manual middleware.

But sometimes, we think it is after a crash and restart (hosted on DigitalOcean app platform) we sometimes get served a plain text webpage instead of the actual full site on one or multiple locales, but there is always some locale that work. As an example, that is the worst case for us. The locale "nb" is not loading properly, but the locale "en" is working. Which is awful to happen as most of our customers will have Norwegian locale. The only way to fix it, is to restart or in DigitalOcean terms "deploy". 

We've only found one instance of another person asking about this issue: [link to stackoverflow question](https://stackoverflow.com/questions/76466634/nextjs-serves-some-text-instead-of-html#comment135685299_76466634) and we are lost on how to treat this. As a stop gap I might just need to force the page to send me an email every time it restarts and manually check if it is working, as the page is still going to serve something so the regular uptime tracker wont catch it. 

It also does not produce any errors that is logged to Sentry or in console. I could not find an example of our own issue, but it looks almost identical to the example in the stackoverflow post. 

Is this DigitalOcean issue, or is this known elsewhere?

**Comment by gerardnico** [comment, marked as answer]
It seems that your error comes from this line:
```
0:[[["",{"children":["__PAGE__",{}]},"$undefined","$undefined",true]
```

If you look at what `__PAGE__` is, you find issues like that: https://github.com/vercel/next.js/issues/48082
